projects/packages/forms/src/blocks/.eslintrc.js

```diff
@@ -0,0 +1,48 @@
+const loadIgnorePatterns = require( 'jetpack-js-tools/load-eslint-ignore.js' );
```

Comment:
We can probably update the package's `.gitattributes` to exclude the whole `src/blocks/**` directory from the production build, since we'll only need the build files in the stable version of the plugin.

Reply:
Could you elaborate a bit more on the suggestion here?

When I added the `eslintrc.js` in this directory, I tried to mimic the same approach used here: https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/jetpack/extensions/.eslintrc.js

Without this file, the linter was complaining about several things.

Reply:
This `.eslintrc.js` is just fine really. I was suggesting excluding all those raw files from the final build, here:
https://github.com/Automattic/jetpack/blob/update/move-form-blocks/projects/packages/forms/.gitattributes#L24

Reply:
Got it! I added the blocks directory to the `.gitattributes` file.
Let me know if there's anything else I could do to improve things.